check/src/main/scala/rsc/checkscalasig/Settings.scala

```diff
@@ -0,0 +1,38 @@
+package rsc.checkscalasig
+
+import java.io.File.pathSeparator
+import java.nio.file.{Path, Paths}
+import rsc.checkbase.SettingsBase
+
+final case class Settings(
+    cp: List[Path] = Nil,
+    ins: List[Path] = Nil,
+    quiet: Boolean = false
+) extends SettingsBase
+
+object Settings {
+  def parse(args: List[String]): Either[List[String], Settings] = {
+    def loop(
+        settings: Settings,
+        allowOptions: Boolean,
+        args: List[String]): Either[List[String], Settings] = {
+      args match {
+        case "--" +: rest =>
```

Comment:
I would prefer using an arg parsing library here both to show usage and for readability.  I can't easily tell what the use case for passing -- as an arg is but it appears to short circuit later options.

Reply:
That will be part of a larger migration to a real parsing library.